core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -6287,6 +6287,14 @@ public Sql schema(CalciteAssert.SchemaSpec schemaSpec) {
     assertThat(toSql(root, DatabaseProduct.CALCITE.getDialect()), isLinux(expectedSql));
     assertThat(toSql(root, DatabaseProduct.SNOWFLAKE.getDialect()), isLinux(expectedSF));
   }
+
+  @Test public void testInstr() {
```

Comment:
change test name, this is not a test for Instr()

Reply:
testREGEXP_INSTR() test is okay???